content/docs/started/getting-started.md

```diff
@@ -66,11 +66,12 @@ Download, set up, and deploy (If you prefer to work from source code, feel free
 1. Run the following commands to setup and deploy Kubeflow:
     
     ```
-    ${KUBEFLOW_SRC}/scripts/kfctl.sh init ${KFAPP} --platform none
+    ${KUBEFLOW_SRC}/scripts/kfctl.sh init ${KFAPP} --platform none --zone ${ZONE}
```

Comment:
Is it always essential to add a zone? Looking at issue [kubeflow/kubeflow/1914](https://github.com/kubeflow/kubeflow/issues/1914) I think it may be OK to use the default zone in most cases.

If use of the default zone is good for most cases, then it's best to keep the init command simple. You could mark the zone parameter as optional, and explain the circumstances under which people need to include the parameter.